proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -259,6 +265,22 @@ message QueryConsumerChainOptedInValidatorsResponse {
   repeated string validators_provider_addresses = 1;
 }
 
+message QueryConsumerChainConsumerValidatorsRequest {
+  string chain_id = 1;
+}
+
+message QueryConsumerChainConsumerValidator {
```

Comment:
We could potentially have used `ConsumerValidator` from [here](https://github.com/cosmos/interchain-security/blob/main/proto/interchain_security/ccv/provider/v1/provider.proto#L325) instead of creating a new `message`. However, `ConsumerValidator` has the `provider_address` as a `bytes` so it would not parse in a nice way when showing it as part of the response.